src/App.js

```diff
@@ -92,13 +92,18 @@ class App extends Component {
   }
 
   render() {
-    return (
-      <div>
-        <Header user={this.state.user} />
-        <img src={logo} alt="logo" />
-        <Footer />
-      </div>
-    );
+    const isLoggedIn = !!this.state.user;
```

Comment:
did you change the related code where `state.user` is set as _empty object_ in place of _null_ value?

Reply:
Now yes!

Reply:
👍  Thanks!